news/_posts/2020-01-07-critical-security-vulnerability-in-prestashop-modules.md

```diff
@@ -56,6 +58,7 @@ XsamXadoo_Bot.php | `0890e346482060a1c7d2ee33c2ee0415` or `b2abcadb37fdf9fb666f1
 XsamXadoo_deface.php | `05fb708c3820d41c95e34f0a243b395e`
 0x666.php | `edec4c4185ac2bdb239cdf6e970652e3`
 f.php | `45245b40556d339d498aa0570a919845` 
+Xsam_Xadoo.html | 
```

Comment:
@ttoine Where did you get this info? Would it be possible to get an md5 digest of the file?

Reply:
https://www.prestashop.com/forums/topic/1012095-hack-prestashop-avec-xsamxadoo-bot/?do=findComment&comment=3187723

Reply:
BTW, I think that we should remove the md5. This is just confusing for most people and the name of the file is quite enough.

Reply:
Okay then